### PR TITLE
Changes in the ModelValidator script

### PR DESCRIPTION
**Pull request by Antonior26**

### CODE REVIEW

File: bin/ModelValidator [hunk 1]
```diff
@@ -23,17 +23,21 @@ class option_parser:
...
 
 
-def validation(json_file, schema, version):
+def validation(json_file, schema, version, package):
     if version == 'latest':
-        protocol = importlib.import_module('protocols.GelProtocols')
+        protocol = importlib.import_module('protocols.'+ package)
     else:
-        protocol = importlib.import_module('protocols.GelProtocols_' + version.replace('.', '_'))
+        protocol = importlib.import_module('protocols.' + 'package' + version.replace('.', '_'))
```
Comment (priesgo) [inline bin/ModelValidator]: Are we sure you want to use the string 'package' and not the variable package???

Comment (Antonior26) [inline bin/ModelValidator]: :(

Comment (Antonior26) [inline bin/ModelValidator]: now?

File: setup.py [hunk 2]
```diff
@@ -20,7 +20,7 @@
     version=VERSION,
     packages=find_packages(),
     include_package_data=True,
-    scripts=['scripts/ModelValidator'],
+    scripts=['bin/ModelValidator'],
```
Comment (priesgo) [inline setup.py]: does this reference exist?

Comment (Antonior26) [inline setup.py]: it has been renamed scripts->bin

Comment (priesgo) [inline setup.py]: yes it does ...

File: bin/ModelValidator [hunk 3]
```diff
@@ -23,17 +23,21 @@ class option_parser:
...
 
 
-def validation(json_file, schema, version):
+def validation(json_file, schema, version, package):
     if version == 'latest':
-        protocol = importlib.import_module('protocols.GelProtocols')
+        protocol = importlib.import_module('protocols.'+ package)
     else:
-        protocol = importlib.import_module('protocols.GelProtocols_' + version.replace('.', '_'))
+        protocol = importlib.import_module('protocols.' + package + version.replace('.', '_'))
```
Comment (priesgo) [inline bin/ModelValidator]: we may be missing an underscore between package and version:
protocol = importlib.import_module('protocols.' + package + '_' + version.replace('.', '_'))

**Pull request merged**